openrpc/src/examples/Transactions.json

```diff
@@ -107,5 +107,168 @@
             "latestLedger": 2553978,
             "latestLedgerCloseTime": "1700159337"
         }
+    },
+    "GetTransactionsRequest": {
+        "name": "getTransactionsRequest",
+        "value": {
+            "startLedger": 1888539,
+            "pagination": {
+                "limit": 5
+            }
+        }
+    },
+    "GetTransactionsRequestUsingCursor": {
+        "name": "getTransactionsRequestUsingCursor",
+        "value": {
+            "pagination": {
+                "cursor": "8111217537191937",
+                "limit": 5
+            }
+        }
+    },
```

Comment:
There is something particular about either a) OpenRPC in general, or b) the plugin/scripts we're using where using an `example` object for the request part of an example pairing works a little funky (screenshot below, where there's a superfluous intermediate object inside the `params` object in the request).

What you'll want to do is remove these request example objects, and create some examples for the individual parameters themselves, like `PaginationLimitTwo` in the `examples/Events.js` file (we're likely better off shuffling some stuff into an `examples/Pagination.js` file?). Then, add those parameter examples to the request in the example pairing object you've made, like in `NativeTransferEventsExample` in `EventsPairs.json`. (wow, that seems clear as mud hehe)

![Screenshot 2024-05-31 at 12 11 41 PM](https://github.com/stellar/stellar-docs/assets/2024293/9faaab53-5ecb-4117-bccf-9702280d69a6)

Reply:
Ahh I did not see this. Thanks for spotting this issues. I will refactor it according to your suggestions.